notebooks/official/training/pytorch_gcs_data_training.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>Drop the first two steps: that's part of the setup</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Done.</p>